API/Controllers/ProjectController.cs

```diff
@@ -550,6 +567,23 @@ public async Task<IActionResult> UpdateProject(int projectId, [FromBody] Project
                 }
             }
 
+            foreach(int projectResourceImageId in projectResource.ImageIds)
+            {
+                Models.File image = await fileService.FindAsync(projectResourceImageId);
+                if(image == null)
+                {
+                    ProblemDetails problem = new ProblemDetails
+                                             {
+                                                 Title = "Image was not found.",
+                                                 Detail = "The specified image was not found while creating project.",
+                                                 Instance = "BBE862F5-0F88-478A-92EF-6864860ACEC5"
+                                             };
+                    return BadRequest(problem);
+                }
+
+                project.Images.Add(image);
+            }
+
```

Comment:
Is there a possibility to also remove images from a project?

Reply:
Removing happens within the update method. When you update with less images, the other will automatically be removed